client/src/endpoint/account.rs

```diff
@@ -267,3 +268,111 @@ mod transactions_tests {
         );
     }
 }
+
+/// An endpoint that accesses all the effects that changed a given account
+#[derive(Debug, Clone)]
+pub struct Effects {
+    id: String,
+    cursor: Option<String>,
+    order: Option<Order>,
+    limit: Option<u32>,
+}
+
+impl Effects {
+    /// Returns a new endpoint for account effects. Hand this to the client in order
+    /// to request effects for a specific account.
+    pub fn new(id: &str) -> Self {
+        Self {
+            id: id.to_string(),
+            cursor: None,
+            order: None,
+            limit: None,
+        }
+    }
+
+    /// Starts the page of results at a given cursor
```

Comment:
We've been adding doc-tests to these. Would you mind adding those in?

Reply:
I can pretty much just mirror what we are doing with the transactions one right?

Reply:
@david-pm you can see some examples here - https://github.com/kbacha/stellar-sdk/blob/master/client/src/endpoint/transaction.rs#L34

Reply:
Yep, cause you'll need an account id.

Reply:
no clue what i'm doing haha
i just used the transaction endpoint to get an account_id since i saw us do that earlier but i should be able to get it from the effects endpoint shouldn't i?

Reply:
@david-pm yeah I think this is good/right. Basically for these tests, the only way to be sure you get a valid account ID without hardcoding it would be to query for transactions and get the account ID from the transaction.